cmd/vic-machine/create/create.go

```diff
@@ -1091,7 +1091,12 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(c.certPath, 0700)
+	files, err := ioutil.ReadDir(c.certPath)
+	if len(files) > 0 {
```

Comment:
I looked at the certificate validation code, which looks not bad. 
My opinion is that we'd better to stop the installation after a failed validation, instead of here. But might need @andrewtchin comments.